### ISSUE TITLE
'Analysis' sidebar option in gene detail page in Tripal3 

### ISSUE THREAD

**Issue by songtaogui** [tripal/tripal#987]
Hi,

In tripal2  there is an 'Analysis' sidebar option in the gene detail page:

>![TIM图片20190814111642](https://user-images.githubusercontent.com/14274330/62992168-02e4fd00-be85-11e9-87bc-bcbd87d2998e.png)

which shows analysis related to this gene feature:

![image](https://user-images.githubusercontent.com/14274330/62992275-6d963880-be85-11e9-836d-40a2a9ffb545.png)

However, in tripal3, this sidebar was disappeared.

So how could I add an 'analysis' sidebar to Tripal 3 gene page?

Thank you

Songtao Gui

**Comment by spficklin** [comment]
Yes, in Tripal v3 there is no field that connects features with analyses in which those features are used. I'm not sure if we missed it or if it was purposely excluded. I'm going to tag this issue as "Discussion" to get input from the @tripal/project-managent-commitee-pmc.

For some content types that link to Chado tables with an `analysis_id` column in the table, there should be a field for the analysis, but for those content types that are derived from the `feature` table of Chado, you're right, it's not there.

**Comment by bradfordcondon** [comment]
I'd welcome this as a contribution.  If we decided it didn't fit into core, i think it would fit nicely in [manage-analyses](https://github.com/statonlab/tripal_manage_analyses).

**Comment by laceysanderson** [comment]
I'd also welcome this as a contribution. Depending on the design, it may fit nicely into core.

Note: with the switch PMC development focus to Tripal 4 there is no current plans by the PMC to add this functionality.

**Comment by spficklin** [comment]
It has been too long since we looked at this issue and it's not a hard one to deal with.   I'm going to mark this as something to work on in one of our weekly Monday morning codefest

**Comment by laceysanderson** [comment]
Moving this over to Tripal 4 as it should not be done in Tripal 3.

**Comment by imnuvi** [comment]
Hi folks, is this issue open to be picked up? I would love to contribute code to Tripal on any issue that requires help. Thanks!

**Comment by dsenalik** [comment]
We would welcome your help! Thank you for looking at an issue that has been neglected for so long, however this was implemented for Tripal 3 by creating this module https://github.com/tripal/chado_linker_fields which provides the `chado_linker__analysis` field which for Tripal 3 will provide the information requested in this issue.
We will be needing this field for Tripal 4, issue #1381, however work on linking fields is paused while we resolve issues with chado storage needed to make these fields work.
Issue #1414 lists fields we are wanting help with, some have been completed, but I think some still need to be claimed

**Comment by dsenalik** [comment]
An easy field to start with might be `sep__protocol` - it has limited usage, but should be "easy"

**Comment by imnuvi** [comment]
sure, will check out #1414 Thanks

**Comment by laceysanderson** [comment]
Stale and specific to Tripal 3
